build.rs

```diff
@@ -0,0 +1,56 @@
+use rerun_except::rerun_except;
+
+use std::env;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+static BOEHM_REPO: &str = "git://github.com/ivmai/bdwgc.git";
+static BOEHM_ATOMICS_REPO: &str = "git://github.com/ivmai/libatomic_ops.git";
+static BOEHM_DIR: &str = "bdwgc";
+static POINTER_MASK: &str = "-DPOINTER_MASK=0xFFFFFFFFFFFFFFF8";
+
+fn run<F>(name: &str, mut configure: F)
+where
+    F: FnMut(&mut Command) -> &mut Command,
+{
+    let mut command = Command::new(name);
+    let configured = configure(&mut command);
+    println!("Executing {:?}", configured);
+    if !configured.status().is_ok() {
+        panic!("failed to execute {:?}", configured);
+    }
+    println!("Command {:?} finished successfully", configured);
+}
+
+fn main() {
+    rerun_except(&["gc_tests/tests/*.rs"]).unwrap();
+
+    let mut bdwgc = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
```

Comment:
Would it be better to put this in `OUT_DIR` so that it doesn't clutter up the repo while it's being built? I also suspect that if/when this is packaged, sticking things in `OUT_DIR` might be safer, but I'm not really sure.

Reply:
Yes. I think we have something a lot better now.